akka-stream/src/main/scala/akka/stream/scaladsl/Flow.scala

```diff
@@ -1460,12 +1460,41 @@ trait FlowOps[+Out, +Mat] {
    *
```

Comment:
We should make it even more clear that extrapolate is always applied, and "if you want elements to flow through unchanged when upstream is keeping up with downstream, use [[link to the new method]]"

Reply:
+1, see later comment.